ingestion/tests/unit/topology/database/test_athena.py

```diff
@@ -0,0 +1,410 @@
+            displayName="s3_local",
+            deleted=False,
+            inherited=None,
+            href=Href(
+                root=AnyUrl(
+                    "http://localhost:8585/api/v1/services/storageServices/dd91cca3-cc54-4776-9efa-48f845cdfb92",
+                )
+            ),
+        ),
+        parent=None,
+        children=None,
+        dataModel=ContainerDataModel(
+            isPartitioned=False,
+            columns=[
+                Column(
+                    name=ColumnName("CUSTOMERID"),
+                    displayName="CUSTOMERID",
+                    dataType=DataType.INT,
+                    arrayDataType=None,
+                    dataLength=None,
+                    precision=None,
+                    scale=None,
+                    dataTypeDisplay="INT",
+                    description=None,
+                    fullyQualifiedName=FullyQualifiedEntityName(
+                        's3_local.awsdatalake-testing."dbt-testing/mayur/customers.csv".CUSTOMERID'
+                    ),
+                    tags=None,
+                    constraint=None,
+                    ordinalPosition=None,
+                    jsonSchema=None,
+                    children=None,
+                    profile=None,
+                    customMetrics=None,
+                ),
+            ],
+        ),
+        prefix="/dbt-testing/mayur/customers.csv",
+        numberOfObjects=2103.0,
+        size=652260394.0,
+        fileFormats=[FileFormat.csv],
+        serviceType=StorageServiceType.S3,
+        followers=None,
+        tags=None,
+        changeDescription=None,
+        deleted=False,
+        retentionPeriod=None,
+        extension=None,
+        sourceUrl=SourceUrl(
+            "https://s3.console.aws.amazon.com/s3/buckets/awsdatalake-testing?region=us-east-2&prefix=dbt-testing/mayur/customers.csv/&showversions=false"
+        ),
+        fullPath="s3://awsdatalake-testing/dbt-testing/mayur/customers.csv",
+        domain=None,
+        dataProducts=None,
+        votes=None,
+        lifeCycle=None,
+        sourceHash="22b1c2f2e7feeaa8f37c6649e01f027d",
+    )
+]
+
+MOCK_TABLE_ENTITY = [
+    Table(
+        id=Uuid(UUID("2c040cf8-432d-4597-9517-4794d6142da3")),
+        name=EntityName("demo_data_ext_tbl3"),
+        displayName=None,
+        fullyQualifiedName=FullyQualifiedEntityName(
+            "local_athena.demo.default.demo_data_ext_tbl3"
+        ),
+        description=None,
+        version=None,
+        updatedAt=Timestamp(1717071974350),
+        updatedBy="admin",
+        href=Href(
+            root=AnyUrl(
+                "http://localhost:8585/api/v1/tables/2c040cf8-432d-4597-9517-4794d6142da3",
+            )
+        ),
+        tableType=TableType.Regular,
+        columns=[
+            Column(
+                name=ColumnName("CUSTOMERID"),
+                displayName=None,
+                dataType=DataType.INT,
+                arrayDataType=None,
```

Comment:
I suggest removing all the `None` fields to make the file more concise. This also allows changing the models without needing to change the test files.